content/master/concepts/managed-resources.md

```diff
@@ -635,21 +635,137 @@ my-rds-instance      True    True     my-custom-name       11m
 
 ### Creation annotations
 
-Providers create new managed resources with the
+Providers use three annotations to avoid and detect leaking resources:
+
+* `crossplane.io/external-create-pending`
+* `crossplane.io/external-create-failed`
+* `crossplane.io/external-create-succeeded`
+
+```yaml {label="creation"}
+apiVersion: ec2.aws.upbound.io/v1beta1
+kind: VPC
+metadata:
+  name: my-vpc
+  annotations:
+    crossplane.io/external-name: vpc-1234567890abcdef0
+    crossplane.io/external-create-pending: "2023-12-18T21:48:06Z"
+    crossplane.io/external-create-succeeded: "2023-12-18T21:48:40Z"
+```
+
+{{<hint "tip">}}
+A provider uses a managed resource's `crossplane.io/external-name` annotation to
+know if it needs to create the resource in the external system. The annotation
+uniquely identifies the resource in the external system, such as AWS.
+
+If the provider doesn't save this annotation, it forgets that it created the
+resource. In fact, it doesn't know the resource ever existed and can't manage
+it. Crossplane calls this a leaked resource.
+{{</hint>}}
+
+Some external systems let the provider specify the name of the resource when
+it's created. The provider can generate a name and save it to the
+`crossplane.io/external-name` annotation before it creates the resource. This
+guarantees the provider doesn't leak the resource.
+
+Other external systems don't let the provider specify the name of the resource.
+Instead they generate an unpredictable name and include it in the response to
+the create API call the provider makes.
+
+{{<hint "important">}}
+For this second kind of external system, there is always a risk that a provider
+leaks a newly created resource.
+{{</hint>}}
+
+A provider can't guarantee that it can save the generated name to the
+`crossplane.io/external-name` annotation. If the provider crashes, restarts, or
+can't reach the network after creating the resource it can't save the
+annotation. If this happens, the provider leaks the resource.
```

Comment:
Close. Perhaps it would help to mention that providers are stateless. When a provider starts, it can only go off the state it reads from the API server (i.e. the recorded state of the MR).

So the timeline is more like this:

* T0: POST from provider
* T1: External resource is created
* T2: Provider stops (e.g. crash, restart) before it can save the `crossplane.io/external-name` annotation
* T3: Provider starts
* T4: Provider loads MR from API server

At T4 the provider only knows what it can read from the API server. When it reads the MR, it has no `crossplane.io/external-name` annotation. Or rarely (if the provider was _re-creating_ the resource because someone deleted it using the cloud console) has a stale `crossplane.io/external-name` annotation, pointing to an external resource that no longer exists.

So at this point as far as the provider can tell the MR doesn't exist, and needs to be created.